database/migrations/2022_04_10_210041_add_new_brand_tags2.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddNewBrandTags2 extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('brands', function (Blueprint $table) {
+            $table->integer('tim_hortons')->unsigned()->nullable();
+            $table->integer('wendys')->unsigned()->nullable();
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        //
```

Comment:
missing reverse migrations?

Reply:
We shouldn't be doing reverse migrations anyway, it's more stable like this. I've learned this from others, and the hard way as well, lol.